src/components/image-tooltip/ImageTooltipContent.tsx

```diff
@@ -3,34 +3,45 @@ import classNames from 'classnames';
 
 import './ImageTooltipContent.scss';
 
+type OnImageLoad = () => void;
+
 export type ImageTooltipContentProps = {
     /** A React component representing the image used in the visual tooltip */
     children: React.ReactElement;
     /** A string to be used in the tooltip's paragraph content */
     content: string;
+    /** A callback triggered onLoad for the image element */
+    onImageLoad?: OnImageLoad;
     /** A string to be used in the tooltip's title heading */
     title: string;
 };
 
-function appendTooltipImageChildClass(child: React.ReactElement): React.ReactElement | null {
+function appendTooltipImageChildClass(child: React.ReactElement, onImageLoad?: OnImageLoad): React.ReactElement | null {
```

Comment:
function name is not really accurate now

Reply:
thanks! updating...